DigitalLearningSolutions.Web/Helpers/PromptHelper.cs

```diff
@@ -3,17 +3,18 @@
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using DigitalLearningSolutions.Data.Models.CourseDelegates;
     using DigitalLearningSolutions.Data.Models.CustomPrompts;
     using DigitalLearningSolutions.Data.Models.User;
     using DigitalLearningSolutions.Data.Services;
     using DigitalLearningSolutions.Web.ViewModels.Common;
     using Microsoft.AspNetCore.Mvc.ModelBinding;
 
-    public class CentreRegistrationPromptHelper
+    public class PromptHelper

```

Comment:
This ain't really a helper in that it's not a static class - can we call it PromptsService? Also because there's a `PromptHelper` in the Data project. I'm aware this is in the web project, but no need to move it in this ticket. I'll raise a tech debt ticket.

Reply:
Renamed!